spec/std/markdown/markdown_spec.cr

```diff
@@ -70,6 +70,8 @@ describe Markdown do
   assert_render "```crystal\nHello\nWorld\n```", "<pre><code class='language-crystal'>Hello\nWorld</code></pre>"
   assert_render "Hello\n```\nWorld\n```", "<p>Hello</p>\n\n<pre><code>World</code></pre>"
   assert_render "```\n---\n```", "<pre><code>---</code></pre>"
+  assert_render "````\n---\n````", "<pre><code>---</code></pre>"
+  assert_render "```invisible man```", "<p><code></code><code>invisible man</code><code></code></p>"
```

Comment:
Because there's no inline code fence, so it's an empty inline code block (` `` `), an inline code block with the "invisible man" content and finally another empty inline code block (` `` `)

Reply:
Wow, thank you! I had the same doubt.

Reply:
It's not so much about what's correct, just that the previous result was much, much worse (see first message). And typing like this is apparently quite common.

Reply:
According to this example of the spec: http://spec.commonmark.org/0.12/#example-95
There should just be one `<code>` block, but as you said, the old behavior was worst, maybe it's ok to do ghis in another pr.

Reply:
Markdown code spans can be delimited by any number of backticks, the only requirement is that start and end contain the same amount. So ` ```invisible man``` `should really only produce `<p><code>invisble man</code></p>`.

Reply:
Inline code blocks are not affected by this PR and are outside the scope.

e.g. this is the same before and after.
```crystal
require "markdown"; p Markdown.to_html("a ```b```")'
```
```
"<p>a <code></code><code>b</code><code></code></p>"
```

I added a TODO for the spec.